doc/source/templates/configs/anyscale_cluster_env.yaml

```diff
@@ -3,4 +3,4 @@
 #   docker_image: anyscale/ray-ml:latest-py39
 #   ray_version: nightly
```

Comment:
Also, can we use the byod since it has the `latest` alias?

```
#   docker_image: anyscale/ray-ml:latest-py39-gpu
#   ray_version: latest
```

Reply:
Update from discussion with Sofian:
- Product does not support "anonymous" cluster envs. So, if we do a BYOD solution, then every new template they create will show up as a new cluster env in their cluster envs listing page.
- So, we'll start with base images build ids.
- The product code-path already supports BYOD, so it would be easy to transition.